screens/NewConversation/NewConversation.tsx

```
@@ -1,164 +1,169 @@
+import { sendMessage } from "@/features/conversation/hooks/use-send-message";
+import { useNavigation } from "@react-navigation/native";
+import { Button } from "@/design-system/Button/Button";
+import { stylesNewChat } from "./newChat.styles";
+
+/**
+ * Screen shown for when user wants to create a new Chat.
+ *
+ * User can search for peers, create a new group, create a dm,
+ * or navigate to an existing dm.
+ */
+export default function NewConversation({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)